src/common/prntbase.cpp

```diff
@@ -2095,7 +2109,8 @@ bool wxPrintPreviewBase::RenderPage(int pageNum)
 
     if (!m_previewBitmap)
     {
-        m_previewBitmap = new wxBitmap(pageRect.width, pageRect.height);
+        m_previewBitmap = new wxBitmap;
+        m_previewBitmap->CreateWithLogicalSize(pageRect.width, pageRect.height, m_previewCanvas->GetContentScaleFactor());
```

Comment:
It would be useful to have a comment here explaining why the page rect is in logical size rather than in DIPs because this is not really obvious.

Reply:
This can be made to work with DIPs in pageRect in theory, but this was meant more like a proof-of-concept, so feel free to modify it.

Reply:
Modifying it is simple, it's testing it that takes time...